src/test/java/website/frontrow/board/MoverTest.java

```diff
@@ -29,7 +29,8 @@
 {
     @SuppressWarnings("checkstyle:visibilitymodifier")
     protected Point m;
-    private Level emptyLevel = new Level(new ArrayList<>(), new ArrayList<>(), new Grid<>(0, 0));
+    @SuppressWarnings("checkstyle:visibilitymodifier")
```

Comment:
I can understand the reason for this, but an explicit comment explaining reasoning would be nice.

< documentation